wolfcrypt/src/asn.c

```diff
@@ -13743,190 +13746,99 @@ static int MatchBaseName(int type, const char* name, int nameSz,
 
 static int ConfirmNameConstraints(Signer* signer, DecodedCert* cert)
 {
+    const byte nameTypes[] = {ASN_RFC822_TYPE, ASN_DNS_TYPE, ASN_DIR_TYPE};
+    long unsigned int i;
```

Comment:
Please use `int` and cast below `(int)sizeof(nameTypes)`

Reply:
I used int at first (because of integer overflow) but I converted to unsigned int to to silence the compiler. Casting to int is a much better solution (C programming is not part of my daily job)

Reply:
done